editor/src/components/editor/actions/actions.tsx

```diff
@@ -4701,7 +4711,11 @@ export const UPDATE_FNS = {
         newSelectedViews.push(newPath)
       }
 
-      const newUID = generateUidWithExistingComponents(editor.projectContents)
+      const existingUids = new Set(getAllUniqueUids(editor.projectContents).uniqueIDs)
+
+      const newUID = generateConsistentUID('zzz', existingUids)
```

Comment:
`'zzz'`?

Reply:
updated it to a more human-readable string in https://github.com/concrete-utopia/utopia/pull/4346/commits/7ca47f68755550aa1bf8b42e15ab3fb2288488ba